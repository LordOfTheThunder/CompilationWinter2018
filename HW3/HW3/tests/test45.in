struct potato {
	int a;
	bool b;
	byte c;
	int d;
};
int PutToPotato(int a, bool b, byte c, int d, struct potato pot) {
	pot.a = a;
	pot.b = b;
	pot.c = c;
	pot.d = d;
}

int ReturnIntFunc() {
	int a = 3;
}

void callPutPotato() {
	struct potato pot;
	PutToPotato(UndefFunc(), false, 33b, 22b, pot);
}





void main() { print("I hate this course"); }



