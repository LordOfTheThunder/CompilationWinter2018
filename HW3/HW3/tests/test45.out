---end scope---
a INT -8
b BOOL -7
c BYTE -6
d INT -5
pot struct potato -4
---end scope---
a INT 0
line 20: function UndefFunc is not defined
